internal/impl/packages.go

```
@@ -390,6 +392,10 @@ func (d *Devbox) syncPackagesToProfile(ctx context.Context, mode installMode) er
 				return err
 			}
 			pending = append(pending, pkg)
+		} else if f, _ := pkg.FlakeInstallable(); d.pluginManager.PathIsInVirtenv(f.Ref.Path) {
```

lets not ignore the error here? `f, _ := pkg.FlakeInstallable();`